README.md

```diff
@@ -279,6 +279,8 @@ The following is the progress report:
 | &#x2705; [#90][] | `singleton`        | `singleton`     |                      | |
 |                  | `skip`             | `skip`          |                      | |
 |                  | `skipWhile`        | `skipWhile`     | `skipWhileAsync`     | |
+|                  |                    |                 | `skipWhileInclusive` | |
+|                  |                    |                 | `skipWhileInclusiveAsync` | |
```

Comment:
Not sure we need those, but if there's a use-case for `takeWhileIncl`... Anyway, this way I won't forget ;).

Reply:
👍 but see naming thought on the Issue